www/templates/html/SiteMaster/Core/Auditor/QATestController.tpl.php

```diff
@@ -0,0 +1,25 @@
+<p>
+    URL To Test: <?php echo $context->url ?>
+</p>
+<?php if (!$context->current_user): ?>
+    <p>We were not able to find a previous scan for this page.  You must log in to continue.</p>
+<?php elseif (!$context->site): ?>
+    <p>
+        We don't know about this site.  Please add it and we will scan it!
+        <a href="<?php \SiteMaster\Core\Config::get('URL') ?>/sites/add/" class="button wdn-button">Register the site now</a>
```

Comment:
If you're worried about too many sites being created (because you're not starting at the domain level), perhaps a new feature should be created that will site admins to "merge" (parent) sites.